docs/matrix-hal/examples/everloop.md

```diff
@@ -0,0 +1,499 @@
+<h2 style="padding-top:0">Everloop</h2>
+<h4 style="padding-top:0">HAL Example</h4>
+
+### Device Compatibility
+
+<img class="creator-compatibility-icon" src="../../img/creator-icon.svg">
+<img class="voice-compatibility-icon" src="../../img/voice-icon.svg">
+
+## Overview
+
+The Everloop interface supports:
+
+- Setting the RGBW colors for each individual LED.
+
+## Code Examples
+
+Function references can be found [here](/matrix-hal/reference/everloop).

```

Comment:
Add text similar to `Below are examples of different ways to interface with the Everloop in MATRIX HAL.` before you mention the function references. Each example should probably follow this